functions/nodejs-apps.bash

```diff
@@ -88,10 +88,14 @@ frontail_setup() {
 
   if ! cond_redirect frontail_download "/opt"; then echo "FAILED (download)"; return 1; fi
   cd /opt/frontail || (echo "FAILED (cd)"; return 1)
-  if ! cond_redirect npm install --force -g; then echo "FAILED (install)"; return 1; fi
```

Comment:
npm install: 
> Is it really a good idea to omit --force ?

Reading the docs of [npm install](https://docs.npmjs.com/cli/v8/commands/npm-install) would help: `The -f or --force argument will force npm to fetch remote resources even if a local copy exists on disk.`
So, there is no real reason to either use it or not use it, except that a user explicitely installs frontail from the menu. In this case, not using `--force` would use the local npm cache and if the version required by the package-lock is in the cache, it uses the local version. When using `--force`, npm downloads even if the same version is in the cache, which is unneccassary traffic.

> The probability of npm breaking changes breaking frontail is rather low,

Talking about this, we are at the `npm update` and `npm audit` commands. I agree that the probability is low, but it is neither zero nor is that the reason why frontail installation would fail.

In issue #1743, frontail fails to install, because there is an unfixed vulnerability and the exit code 1 from `npm audit` is not ignored. 

> Adding npm audit fix --force is probably a good idea, too, see BATS log.

It could only introduce breaking changes, but it would not solve our problem. Keep in mind that the installation in #1743 fails even though `npm audit --force` is used. The force option obviously is unable to fix all vulnerabilities, but instead can introduce breaking changes.

Reply:
Well, then. Is there a "true" --force npm option that does what I have been thinking so far it does, i.e. install without asking for confirmation ?

Reply:
AFAIK, there is nothing like that.
But I am not sure why it is needed, you said that there are cases where a npm package requires manual installation. I am super experienced with npm, but I can not remember any case where I installed a package using `npm install` and npm asked my for anything.

IMO, this should be good to go as it currently is.

FYI: I also found the reason why amd64-bats was failing: The `--package-lock-only` option, but I am wondering that it was a problem.